packages/office-addin-sso/src/commands.ts

```diff
@@ -17,6 +18,19 @@ export async function configureSSO(manifestPath: string) {
   if (process.platform !== "win32" && process.platform !== "darwin") {
     console.log(chalk.yellow(`${process.platform} is not supported. Only Windows and Mac are supported`));
     return;
+  } else if (applicationDataConfigured(manifestPath)) {
+    console.log(
+      chalk.yellow("Project has already been at least partially configured.  Can't setup new configuration completely.")
+    );
+    const question = {
+      message: `Continue anyway?`,
+      name: "didUserConfirm",
+      type: "confirm",
+    };
+    const answer = await inquirer.prompt([question]);
+    if (!(answer as any).didUserConfirm) {
```

Comment:
The example in the answers listed there shows that calling a method (and I presume a property) is not valid for "unknown" type . . . which is exactly what I'm trying to do here.  

This was actually copied from another location in our code.  I will double check if there is not a better way to do this.

Reply:
I guess just changing `any` to `unknown` here should work, but I didn't test it.
Maybe the rest of the code still has a lot of `any`s, I just think unknown would be better as it is more restricted. 
But anyway, it is a very minor deal.

Reply:
It is better to use `unknown` rather than `any` but even better is to have actual type info. I believe the if statement listed here can be written without using unknown or any. What is the type of the answer variable?

Reply:
The 'prompt' method returns 'any' type.  I figured out I don't need the cast and have a local change to remove it.